etc/workers.conf

```diff
@@ -120,4 +120,3 @@ reference-list:
     taglib-gm: 1            # loads tags using a worker (performance thing)
     talklib-gm: 1           # loads comments using a worker (performance thing)
     t-memlimit: 1           # for running certain tests
-    userpic-resize-gm: 1    # resizes userpics
```

Comment:
Out of curiosity, what's the behavior if workers.conf includes a nonexistent worker?  Will it complain about trying to start a nonexistent worker, or silently fail?